client/src/pages/NotFound/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+  font-size: 1.6rem;
+  padding: 10vw;
+  text-align: center;
+  color: ${p => p.theme.color.secondary};
+`;
+
+const NotFound = () => {
+  return (
+    <Wrapper>
+      <p>Nothing to see here.</p>
+      <p>We're pretty good at our jobs, so this is probably your fault.</p>
+    </Wrapper>
+  );
+};
```

Comment:
No need to change, but if you have no logic in the component you can just do this
```suggestion
const NotFound = () => (
    <Wrapper>
      <p>Nothing to see here.</p>
      <p>We're pretty good at our jobs, so this is probably your fault.</p>
    </Wrapper>
  );
```